### ISSUE TITLE
There is no video in Windows 7 VFW 

### ISSUE THREAD

**Issue by HawkQing** [opencv/opencv#5569]
There is a bug that the image size of the output format is zero, this will cause the video frame can't be decompressed. The solution is adding following code in static BITMAPINFOHEADER icvBitmapHeader( int width, int height, int bpp, int compression = BI_RGB ) in cap_vfw.cpp:
"bmih.biSizeImage = width*height*bpp / 8;". I have tested it and passed.

**Comment by mshabunin** [comment]
@HawkQing, can you please create pull request with the proposed fix?

**Comment by jlubawy** [comment]
I stumbled on this while investigating an issue with video decompression using the Surface 4 camera driver and I don't think this is actually a problem. According to the MSDN documentation the biSizeImage field ["may be set to zero for BI_RGB bitmaps"](https://msdn.microsoft.com/en-us/library/dd183376(v=vs.85).aspx?f=255&MSPPError=-2147217396) which is what is being created in this case. Just to be sure I tried changing it in my code and it didn't change anything for me.
